.yamato/upm-ci-publish-github-release.yml

```diff
@@ -16,28 +16,42 @@ publish_com.unity.webrtc_github_release:
   name: Publish com.unity.webrtc to Github Release
   agent:
     type: Unity::VM
-    image: package-ci/win10:stable
+    image: package-ci/ubuntu:stable
     flavor: b1.large
+  variables:
+    GIT_TAG: Foo
```

Comment:
Is this variables needed ?

Reply:
I see, `GIT_TAG` value is `Foo`, is this correct?
I think `GIT_TAG` value need to set webrtc-version string (example `2.4.0-exp.1`).

Reply:
Usually, we need to set the parameter on Yamato UI.
We don't have to change this line for every release.
![image](https://user-images.githubusercontent.com/1132081/116328344-c8abc280-a803-11eb-903e-e1507d7fe0f8.png)

Reply:
OK, `Foo` may not suitable for us, so how about `You need to change this parameter on Yamato UI`?
Is it more clear than `Foo`?

Reply:
> 
```suggestion
  variables:
    GIT_TAG: You need to change this parameter on Yamato UI
```